config/analytics.yml

```diff
@@ -268,6 +268,7 @@ shared:
     - system_admin
     - updated_at
     - welcome_email_sent_at
+    - searchable
```

Comment:
I'm not sure this is useful for sending to bigquery - i see we added the school `searchable` to the blocklist.